test/e2e/cypress/integration/clusters_overview.js

```diff
@@ -0,0 +1,306 @@
+import {
+  allClusterNames,
+  allClusterIds,
+  clusterIdByName,
+} from '../fixtures/clusters-overview/available_clusters';
+
+context('Clusters Overview', () => {
+  const availableClusters = allClusterNames();
+  const availableClustersId = allClusterIds();
+  before(() => {
+    cy.resetDatabase();
+    cy.loadScenario('healthy-27-node-SAP-cluster');
+    cy.loadChecksCatalog('checks-catalog/catalog.json');
+    cy.loadChecksResults(
+      'clusters-overview/checks_results_critical.json',
+      '04b8f8c21f9fd8991224478e8c4362f8'
+    );
+    cy.loadChecksResults(
+      'clusters-overview/checks_results_warning.json',
+      '4e905d706da85f5be14f85fa947c1e39'
+    );
+    cy.loadChecksResults(
+      'clusters-overview/checks_results_passing.json',
+      '9c832998801e28cd70ad77380e82a5c0'
+    );
+
+    cy.visit('/');
+    cy.navigateToItem('Clusters');
+    cy.url().should('include', '/clusters');
+  });
+
+  describe('Registered Clusters should be available in the overview', () => {
+    it('should show 9 of the 9 registered clusters with default pagination settings', () => {
+      cy.get('.tn-clustername').its('length').should('eq', 9);
+    });
+    it('should show 9 as total items in the pagination controls', () => {
+      cy.get('.pagination-count').should('contain', '9 items');
+    });
+    it('should have 1 pages', () => {
+      cy.get('.page-item').its('length').should('eq', 3); // We add +2 to the page count because of the first and last page
+    });
+    describe('Discovered clusternames are the expected ones', () => {
+      availableClusters.forEach((clusterName) => {
+        it(`should have a cluster named ${clusterName}`, () => {
+          cy.get('.tn-clustername').each(($link) => {
+            const displayedClusterName = $link.text().trim();
+            expect(availableClusters).to.include(displayedClusterName);
+          });
+        });
+      });
+    });
+  });
+
+  describe('Health Detection', () => {
+    describe('Health Container shows the health overview of all Clusters', () => {
+      it('should show health status of the entire cluster of 9 hosts with partial pagination', () => {
+        cy.reloadList('clusters', 10);
+        cy.get('.health-container .health-passing').should('contain', 1);
```

Comment:
Maybe we could check that warning and critical are 0 here. Otherwise, the test is a bit redundant

Reply:
I agree this test is a bit useless as it's testing a pagination value of 10 vs 100 where in any case, all of them are going to be shown since we have just 9 clusters. Improving it a bit to have it ready in case we want to add more data to the fixtures, but I think it should be already a good improvement as is for now.